service.go

```diff
@@ -6,12 +6,26 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-	"strings"
+	"sort"
```

Comment:
This file needs the copyright comments at the top.  Other files may be missing this too.

Reply:
Cool, added an issue for it https://github.com/xmidt-org/ancla/issues/30